prepare-build.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
-
-set -e
-
-cd build
-mv static resources
-mv favicon.png resources/
-mv media/* resources/media
-rm -r media
```

Comment:
these as well are changes from #186 to fix images being broken; not required with proper paths